doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_mongo_sink.md

```diff
@@ -314,8 +314,8 @@ If `data_model=dm-by-entity` and `attr_persistence=column` then `NGSIMongoSink`
 | batch\_ttl | no | 10 | Number of retries when a batch cannot be persisted. Use `0` for no retries, `-1` for infinite retries. Please, consider an infinite TTL (even a very large one) may consume all the sink's channel capacity very quickly. |
 | batch\_retry\_intervals | no | 5000 | Comma-separated list of intervals (in miliseconds) at which the retries regarding not persisted batches will be done. First retry will be done as many miliseconds after as the first value, then the second retry will be done as many miliseconds after as second value, and so on. If the batch\_ttl is greater than the number of intervals, the last interval is repeated. |
 | data\_expiration | no | 0 | Collections will be removed if older than the value specified in seconds. The reference of time is the one stored in the `recvTime` property. Set to 0 if not wanting this policy. |
-| collections\_size | no | 0 | The oldest data (according to insertion time) will be removed if the size of the data collection gets bigger than the value specified in bytes. Notice that the size-based truncation policy takes precedence over the time-based one. Set to 0 if not wanting this policy. Minimum value (different than 0) is 4096 bytes. |
-| max\_documents | no | 0 | The oldest data (according to insertion time) will be removed if the number of documents in the data collections goes beyond the specified value. Set to 0 if not wanting this policy. |
+| collections\_size | no | 0 | The oldest data (according to insertion time) will be removed if the size of the data collection gets bigger than the value specified in bytes. Notice that the size-based truncation policy takes precedence over the time-based one. Set to 0 if not wanting this policy. Minimum value (different than 0) is 4096 bytes. More detail in [mongo java driver]( https://mongodb.github.io/mongo-java-driver/3.6/javadoc/com/mongodb/client/model/CreateCollectionOptions.html) |
+| max\_documents | no | 0 | The oldest data (according to insertion time) will be removed if the number of documents in the data collections goes beyond the specified value. Set to 0 if not wanting this policy. More detail in [mongo java driver]( https://mongodb.github.io/mongo-java-driver/3.6/javadoc/com/mongodb/client/model/CreateCollectionOptions.html)|
```

Comment:
We are using version 3.12 (https://github.com/telefonicaid/fiware-cygnus/blob/master/cygnus-common/pom.xml#L100-L102)

Should the link be changed 3.6 -> 3.12?

Reply:
Absolutely yes https://github.com/telefonicaid/fiware-cygnus/pull/2163/commits/d4758845dc5b574de7509fc562b17882ddca7388